molecule/default/tests/test_dashboard.py

```diff
@@ -4,22 +4,43 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
 @pytest.mark.jira('asc-222')
-def test_cinder_service(host):
+def test_cinder_service_on_all(host):
     """Test to verify that cinder service is running on the cinder nodes
 
     Args:
         host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml
     """
+
     if not host.file('/root/openrc').exists:
-        pytest.skip('openrc file not found')
+        raise Exception('openrc file not found')
     elif host.file('/root/openrc').exists:
         cmd = "sudo bash -c \"source /root/openrc; cinder service-list\""
         output = host.run(cmd)
         if output.rc != 0:
-            pytest.skip('cinder client is not available')
+            raise Exception('cinder client is not available')
         elif output.rc == 0:
             assert ("cinder-volume" in output.stdout)
+
+
+# Experimental tess
+def test_cinder_api_status_on_all_host(host):
```

Comment:
Please remove the `_on_all_host` suffix from these tests. The test does not control the scope of the hosts that are passed to it and the test is iterated for individual hosts and is never passed a list.